contributing.md

```diff
@@ -13,40 +13,39 @@
   * [Packaging](#packaging)
 
 
-These instructions are for people who want to download the code and edit it directly. If you are interested in tabcmd but not the code, see [here](Readme.md).
-####To work with tabcmd, you need to have **Python 3.7+** installed.
+## Install Tabcmd
+> [!NOTE]
+> These instructions are for people who want to write code in or using tabcmd. If > you are interested in tabcmd but not the code, see [here](Readme.md).
```

Comment:
I'm having some trouble understanding this line.

Also, I think there's a stray `<` in the 2nd sentence.